### PR TITLE
Added a flag to clear the database upon app start

### PR DESCRIPTION
**Pull request by daika7ana**
Added a new flag -cd, --clear-db to clear pogom.db.

## Description
Made it so the new flag clears the local database or the mysql database prior to adding the new data.
## Motivation and Context
#2249

## How Has This Been Tested?
Tested on my own server under Win10, how it reacts with and without the flag. Everything seems to be working as intended.

## Screenshots (if appropriate):
prior
after
The size can be seen at the bottom and the -cd flag can be seen at the top.

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [x] My code follows the code style of this project.
- [x] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.

### CODE REVIEW

**Comment by jmccorm** [comment]
Hottttt!

**Comment by bdoner** [comment]
Remember that the -D flag can name the database something other that pogom.db. You might want to take that into account.

**Comment by trol4ever** [comment]
Cheers for both your inputs :+1:

File: runserver.py [hunk 1]
```diff
@@ -49,6 +49,11 @@ def start_locator_thread(args):
         logging.getLogger("rpc_api").setLevel(logging.DEBUG)
 
     db = init_database()
+    if args.clear_db:
+        if args.db_type == 'mysql':
+            drop_tables(db)
+        else:
+            os.remove(args.db)
```
Comment (invisiblek) [inline runserver.py]: need to check if this exists first

File: pogom/models.py [hunk 2]
```diff
@@ -335,3 +335,8 @@ def create_tables(db):
     db.connect()
     db.create_tables([Pokemon, Pokestop, Gym, ScannedLocation], safe=True)
     db.close()
+
+def drop_tables(db):
+    db.connect()
+    db.drop_tables([Pokemon, Pokestop, Gym, ScannedLocation], safe=True)
```
Comment (invisiblek) [inline pogom/models.py]: this throws warnings if these don't exist. not the end of the world but if there's a way to avoid that, it'd be preferrable

**Comment by invisiblek** [comment]
rebase and a bit of error handling and this will be good

**Pull request merged**